sorc/ncep_post.fd/MPI_FIRST.f

```diff
@@ -82,20 +82,25 @@ SUBROUTINE MPI_FIRST()
               pp10cb, ti
       use soil, only:  smc, stc, sh2o, sldpth, rtdpth, sllevel
       use masks, only: htm, vtm, hbm2, sm, sice, lmh, gdlat, gdlon, dx, dy, lmv
-      use ctlblk_mod, only: me, num_procs, jm, jsta, jend, jsta_m, jsta_m2,           &
+      use ctlblk_mod, only: me, num_procs, jm, jsta, jend, jsta_m, jsta_m2,ista,iend ,          &
               jend_m, jend_m2, iup, idn, icnt, im, idsp, jsta_2l, jend_2u,            &
               jvend_2u, lm, lp1, jsta_2l, jend_2u, nsoil, nbin_du, nbin_ss,           &
-              nbin_bc, nbin_oc, nbin_su
+              nbin_bc, nbin_oc, nbin_su,                                               &
+              ISTA_M,IEND_M,ISTA_M2,IEND_M2,                                          &
+              ISTA_2L, IEND_2U,IVEND_2U             
```

Comment:
I suggest use lowcase for all variables, which looks more consistent.

ISTA_M,IEND_M,ISTA_M2,IEND_M2, 
ISTA_2L, IEND_2U,IVEND_2U 

replaced to:
ista_m,iend_m,ista_m2,iend_m2,
ista_2l, iend_2u,ivend_2u





    
    
      
    <span class="blob-code-inner blob-code-marker" data-code-marker="+">              ISTA_2L, IEND_2U,IVEND_2U             </span><!--EndFragment-->